server.js

```diff
@@ -4,6 +4,10 @@ var http = require('http')
 
 // Init WS SECRET
 var WS_SECRET
+var ranking = {}
+setInterval(function(){ 
+   ranking = {}   
+}, 86400000);
```

Comment:
By this way, if it cleans all the nodes until 24 hours and if the last catch up was raised the last minute with this solution it will be lost.

Perhaps, may be a good idea, store in the detail of the ranking the date of each catchUp, and accumulate on each emits.

 When you clean the ranking, you should go through the list of cathUp from the oldest to the first still valid and delete them.